src/database/seeders/20190905042406-create-trip-request-seeder.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-unused-vars */
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('TripRequests', [
+    {
+      origin: 'lagos',
+	    destination:'Abj',
+	    reason: 'i want to tour around the city',
+	    departureDate: '12-Dec-2018',
+	    accomodationId: 1,
+	    type: 'one-way',
+      returnDate: '1-Jan-2019',
+      userId: 1,
+      status:'pending',
+      createdAt: new Date(),
+      updatedAt : new Date()
+    },
+    {
+      origin: 'PH',
+	    destination:'Lagos',
+	    reason: 'i want to tour around the city',
+	    departureDate: '12-Dec-2018',
+	    accomodationId: 2,
+	    type: 'two-ways',
+      returnDate: '1-Jan-2019',
+      userId: 2,
+      status:'approved',
+      createdAt: new Date(),
+      updatedAt : new Date()
+    },
+    {
+      origin: 'USA',
+	    destination:'London',
+	    reason: 'i want to tour around the city',
+	    departureDate: '12-Dec-2018',
+	    accomodationId: 3,
+	    type: 'one-way',
+      returnDate: '1-Jan-2019',
+      userId: 2,
+      status:'rejected',
+      createdAt: new Date(),
+      updatedAt : new Date()
+    },
+    {
+      origin: 'lagos',
+	    destination:'Abj',
+	    reason: 'i want to tour around the city',
+	    departureDate: '12-Dec-2018',
+	    accomodationId: 3,
+	    type: 'two-ways',
+      returnDate: '1-Jan-2019',
+      userId: 3,
+      status:'rejected',
+      createdAt: new Date(),
+      updatedAt : new Date()
+    },
+    {
+      origin: 'Abj',
+	    destination:'Lagos',
+	    reason: 'i want to tour around the city',
+	    departureDate: '12-Dec-2018',
+	    accomodationId: 3,
+	    type: 'one-way',
+      returnDate: '1-Jan-2019',
+      userId: 3,
+      status:'approved',
+      createdAt: new Date(),
+      updatedAt : new Date()
+    },
+    {
+      origin: 'Abia',
+	    destination:'Germany',
+	    reason: 'i want to tour around the city',
+	    departureDate: '12-Dec-2018',
+	    accomodationId: 3,
+	    type: 'two-ways',
+      returnDate: '1-Jan-2019',
+      userId: 2,
+      status:'approved',
+      createdAt: new Date(),
+      updatedAt : new Date()
+    },
+    {
+      origin: 'lagos',
+	    destination:'Abj',
+	    reason: 'i want to tour around the city',
+	    departureDate: '12-Dec-2018',
+	    accomodationId: 3,
+	    type: 'one-way',
+      returnDate: '1-Jan-2019',
+      userId: 1,
+      status:'approved',
+      createdAt: new Date(),
+      updatedAt : new Date()
+    },
+  ], {}),
+
+  down: (queryInterface, Sequelize) => queryInterface.bulkDelete('TripRequests', null, {})
+};
```

Comment:
Please, can you align your code well, you can do without `createdAt` and `updatedAt` in your seed data.